skimage/transform/steerable.py

```diff
@@ -0,0 +1,406 @@
+from __future__ import division
+import numpy as np
+from .._shared.utils import assert_nD
+from ..util import img_as_float
+
+
+def build_steerable(image, height=5, nbands=4):
+    """ Construct a Steerable subband decomposition of a gray scale image.
+
```

Comment:
A second, detailed paragraph is needed, describing the algorithm, what it is used for, etc.  Implementation details should be added under Notes.  This also holds for the function below, `recon_steerable`, or at least a See Also added to `build_steerable`.